src/platform_impl/web/device/constants.rs

```diff
@@ -0,0 +1,70 @@
+use super::gamepad::EventCode;
+
+pub const AXIS_LSTICKX: EventCode = EventCode(0);
+pub const AXIS_LSTICKY: EventCode = EventCode(1);
+// pub const AXIS_LEFTZ: EventCode = EventCode(2);
+pub const AXIS_RSTICKX: EventCode = EventCode(3);
+pub const AXIS_RSTICKY: EventCode = EventCode(4);
+// pub const AXIS_RIGHTZ: EventCode = EventCode(5);
+// pub const AXIS_DPADX: EventCode = EventCode(6);
+// pub const AXIS_DPADY: EventCode = EventCode(7);
+pub const AXIS_RT: EventCode = EventCode(8);
+pub const AXIS_LT: EventCode = EventCode(9);
```

Comment:
Why use these EventCodes instead of the Axis struct directly?

Reply:
I do not understand your point 🤔. Do you talk about Axis from GamepadEvent https://github.com/rust-windowing/winit/blob/gamepad-device-events/src/event/device.rs#L156 ?

Reply:
I got it!